src/utilities/zAuctionRouting.ts

```diff
@@ -13,7 +13,8 @@ export const getZAuctionInstanceForDomain = async (
   for (const route of zAuctionRoutes) {
     const match = RegExp(`^${route.uriPattern}`).exec(domainName);
     if (match) {
-      return route.instance;
+      const config = route.config;
+      return zAuction.createInstance(config);
```

Comment:
We don't want to be creating a new instance of zAuction every time we try to access an instance of it.
The reason being that the zAuction SDK instance might have cached data.

This will require some more code changes, but try converting from config->instance when the zns sdk instance gets created. You could lazy instantiate the instance here if you want too (only create the instance when it's used the first time)